mitmproxy/certs.py

```diff
@@ -196,7 +196,7 @@ def load_dhparam(path):
             return dh
 
     @classmethod
-    def from_store(cls, path, basename, key_size):
+    def from_store(cls, path, basename, key_size, passphrase: str = None):
```

Comment:
Let's make this argument bytes right away and call `options.cert_passphrase.encode()` when calling it. The earlier we convert this, the more likely we are to not run into encoding issues. This also removes the slightly ugly `None if passphrase is None else ` :)

```suggestion
    def from_store(cls, path, basename, key_size, passphrase: typing.Optional[bytes] = None):
```

Reply:
Changed it, but we still need one "ugly" if statement in config.py